packages/mc-scripts/src/commands/config-sync.js

```diff
@@ -126,12 +128,35 @@ const configSync = async () => {
     return;
   }
 
-  // TODO: show diff (followup task)
+  const customAppLink = getMcUrlLink(
+    mcApiUrl,
+    fetchedCustomApplication.organizationId,
+    fetchedCustomApplication.application.id
+  );
+
+  // Sanitize SVG to be consistent with the one in MC
+  localCustomAppData.icon = sanitizeSvg(localCustomAppData.icon);
```

Comment:
Maybe we can do that when we parse the config instead of here.

https://github.com/commercetools/merchant-center-application-kit/blob/238bd34a6045fd4cda443585b714ac0689ecd335/packages/application-config/src/transformers.ts#L43